Content/Items/Manabonds/BasicManabond.cs

```diff
@@ -0,0 +1,155 @@
+﻿using StarlightRiver.Helpers;
+using System.Collections.Generic;
+using Terraria.Graphics.Effects;
+using Terraria.ID;
+
+namespace StarlightRiver.Content.Items.Manabonds
+{
+	internal class BasicManabond : Manabond
+	{
+		public override string Texture => AssetDirectory.ManabondItem + Name;
+
+		public BasicManabond() : base("Manabond", "Your minions can store 40 mana\nYour minions siphon 6 mana per second from you untill full\nYour minions spend 6 mana to attack with a magic bolt occasionally") { }
+
+		public override void SafeSetDefaults()
+		{
+			Item.value = Item.sellPrice(gold: 5);
+			Item.rare = ItemRarityID.Blue;
+		}
+
+		public override void MinionAI(Projectile minion, ManabondProjectile mp)
+		{
+			if (mp.timer % 60 == 0 && mp.mana >= 6 && mp.target != null)
+			{
+				mp.mana -= 6;
```

Comment:
I'd like to favor DirectionTo over Normalize(target - center) for readability purposes

Reply:
Furthermore, I see no reason not to use Projectile.NewProjectileDirect() to get the projectile instance instead of the whoAmI.